auth-web/src/components/auth/staff/PPRLauncher.vue

```diff
@@ -0,0 +1,75 @@
+<template>
+  <v-card
+    :class="{ 'product-container': true, 'border-left': addBorder }"
```

Comment:
may this can be handled using css :hover pseudo class
may be like 
.product-container{
&:hover{
border-left: 3px solid $app-blue !important;
}
}

Reply:
Oh good point that's much simpler

Reply:
done